packs/get_live_hosts/actions/get_live_hosts.py

```diff
@@ -0,0 +1,26 @@
+import nmap
+from st2actions.runners.pythonrunner import Action
+
+
+
+class check_live_hosts(Action):  
```

Comment:
Minor style thing - please use camel case for class name (`CheckLiveHostsAction`).